### PR TITLE
Fix for renderer parameter required in Django 2.1

### LINKED ISSUE #98: Not compatible with Django 2.1
I did a basic install and added a `PlainLocationField` to my model, and now the admin crashes with:

```
TypeError: render() got an unexpected keyword argument 'renderer'
```

It seems [on the Django side](https://github.com/django/django/blob/87dc0844a634360182bcb74d491508111ef8a652/django/forms/boundfield.py#L93) it is calling:

```
widget.render(
            name=self.html_initial_name if only_initial else self.html_name,
            value=self.value(),
            attrs=attrs,
            renderer=self.form.renderer,
        )
```

But the `LocationWidget` doesn't accept renderer as a keyword argument.

### PR DESCRIPTION
**Pull request by carlmjohnson**
Fixes #98.

### CODE REVIEW

File: location_field/widgets.py [hunk 1]
```diff
@@ -47,7 +47,11 @@ def render(self, name, value, attrs=None):
         attrs = attrs or {}
         attrs['data-location-field-options'] = json.dumps(self.options)
 
-        text_input = super(LocationWidget, self).render(name, value, attrs)
+        # Django added renderer parameter in 1.11, made it mandatory in 2.1
+        try:
+            text_input = super(LocationWidget, self).render(name, value, attrs=attrs, renderer=renderer)
+        except TypeError:
+            text_input = super(LocationWidget, self).render(name, value, attrs=attrs)
```
Comment (caioariede) [inline location_field/widgets.py]: Can we just pass `renderer` whenever it is not `None`? For example:

```python
super_kwargs = {'attrs': attrs}
if renderer is not None:
    super_kwargs['renderer'] = renderer
text_input = super(LocationWidget, self).render(name, value, **super_kwargs)
```

Comment (carlmjohnson) [reply location_field/widgets.py]: Pushed.

**Comment by caioariede** [comment]
@carlmjohnson thanks man!

**Comment by caioariede** [comment]
@carlmjohnson mind submitting another PR including yourself in the AUTHORS file?

**Comment by alantrick** [comment]
Can we get another pypi release for this?

**Pull request merged**